examples/gno.land/p/demo/mux/query_test.gno

```diff
@@ -0,0 +1,72 @@
+package mux
```

Comment:
btw, not your problem (i'm the author), but I think we should rename `mux` to `router`